include/agl/driver/aglNVNMgr.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <heap/seadDisposer.h>
+#include <hostio/seadHostIONode.h>
+#include "agl/driver/aglGraphicsDriverMgr.h"
+
+namespace agl::driver {
+
+class NVNMgr : GraphicsDriverMgr {
```

Comment:
`public GraphicsDriverMgr`? (or is there a reason for using private inheritance?)